packages/host/app/components/operator-mode/submode-layout.gts

```diff
@@ -191,6 +199,76 @@ export default class SubmodeLayout extends Component<Signature> {
     this.suppressSearchClose = false;
   });
 
+  @cached
+  private get roomResources() {
+    let resources = new TrackedMap<string, RoomResource>();
+    for (let roomId of this.matrixService.rooms.keys()) {
+      resources.set(
+        roomId,
+        getRoom(this, () => roomId),
+      );
+    }
+    return resources;
+  }
+
+  @use private findUnseenMessage = resource(() => {
+    const state: {
+      value: {
+        roomId: string;
+        message: MessageField;
+      } | null;
+      load: () => Promise<void>;
+    } = new TrackedObject({
+      value: null,
+      load: async () => {
+        let lastMessages: Map<string, MessageField> = new Map();
+        for (let resource of this.roomResources.values()) {
+          if (!resource.room) {
+            continue;
+          }
+          let { room } = resource;
+          lastMessages.set(
+            room.roomId,
+            room.messages[room.messages.length - 1],
+          );
+        }
+
+        let lastMessage =
+          Array.from(lastMessages).filter(
+            (lastMessage) =>
+              lastMessage[1] &&
+              !this.matrixService.currentUserEventReadReceipts.has(
+                lastMessage[1].eventId,
+              ),
+          )[0] ?? null;
+        if (lastMessage) {
+          state.value = {
+            roomId: lastMessage[0],
+            message: lastMessage[1],
+          };
+          setTimeout(() => {
+            state.value = null;
+          }, 3000);
+        }
+      },
+    });
+
+    state.load();
```

Comment:
We have used this pattern in the past when we wanted to manually reload the resource by exposing a `load` method, I'm not sure if we need this here